README.md

```diff
@@ -147,7 +147,116 @@ You can also use Docker Compose to run all the components of EvalAI together. Th
     ```
     python scripts/workers/submission_worker.py
     ```
+    
+## Installation Instructions for Windows
```

Comment:
I would recommend you to add this to the docs/ instead of putting it in the README. We don't want to clutter the README file.